scrnaseq_hto.Rmd

```diff
@@ -189,15 +374,15 @@ hto_t_raw_pseudo = hto_t_raw + 1
 hto_t_norm = Seurat::GetAssayData(sc, assay="HTO", slot="data") %>% 
   as.data.frame %>% t %>% as.data.frame
 
-p1 = ggplot(hto_t_raw_pseudo %>% tidyr::pivot_longer(tidyr::everything()), aes(x=name, y=value, fill=name)) + 
+p1 = ggplot(hto_t_raw_pseudo %>% tidyr::gather(key="name"), aes(x=name, y=value, fill=name)) + 
```

Comment:
Out of interest: what is `gather` doing here compared to `pivot_longer`?

Reply:
Went back to `pivot_longer`